runtime/themes/dracula.toml

```diff
@@ -120,6 +120,7 @@
 "ui.virtual.inlay-hint"           = { fg    = "cyan"                                                        }
 "ui.virtual.inlay-hint.parameter" = { fg    = "cyan",            modifiers = ["italic", "dim"]              }
 "ui.virtual.inlay-hint.type"      = { fg    = "cyan",            modifiers = ["italic", "dim"]              }
+"ui.virtual.jump-label"           = { fg    = "pink",            modifiers = ["bold"]                       }
```

Comment:
I just tried out this theme, I suspect this is a "me" problem but wanted to check-in

Should fg be updating the actual text characters colour? 
Using this instead of my custom one and it just seems to flip the characters without additional styles? (I'm using iTerm2 on MacOS, and I may not have that set up properly)

![image](https://github.com/helix-editor/helix/assets/485809/3efb2c62-92b6-4111-b1bb-bc7d34ac32ad)

Reply:
yeah it's supposed to highlight the 2 first characters in pink. Example:
![image](https://github.com/helix-editor/helix/assets/40139584/5a0714ef-82d1-4b11-a660-0ff0cafdcf08)

Reply:
@yo-main Ah okay yeah I can see how that fits the Dracula style well now! Thanks for the screenshot, I'll keep poking around my Terminal's settings then.

I'm using `reversed` in my custom theme, which is pretty in-your-face but makes it clear something has happened (especially with how often I've been pairing with other devs recently)
![image](https://github.com/helix-editor/helix/assets/485809/cf730835-ee73-4187-9630-f592f0b11de4)